### ISSUE TITLE
Value of sockets_used from /api/nodes is always 0

### ISSUE THREAD

**Issue by jgkirschbaum** [rabbitmq/rabbitmq-management#652]
RabbitMQ version: `3.7.10` rabbitmq-server-generic-unix-3.7.10.tar.xz
Erlang version: `21.2.2` erlang-21.2.2-1.el7.centos.x86_64.rpm
Operating system: `CentOS 7` Linux release 7.6.1810 (Core)

When performing a GET request on the HTTP management API `/api/nodes` with following command:

`curl -sSk -X GET -u 'user:pass' 'https://mq-server:8443/api/nodes' | jq '.[].sockets_used'`

The result for `sockets_used` is always `0` for all nodes in the cluster, although there are connected clients.

* With RabbitMQ `3.7.10` and Erlang `21.2.2` the field `sockets_used` is always `0`.
* With RabbitMQ `3.7.10` and Erlang `19.3.6.12` the field `sockets_used` is always `0`.
* With RabbitMQ `3.6.16` and Erlang `19.3.6.12` the field `sockets_used` is set correctly.

The field `Socket descriptors` in the management UI is also `0` for all nodes in the cluster.

**Comment by michaelklishin** [comment]
Thank you for your time.

Team RabbitMQ uses GitHub issues for specific actionable items engineers can work on. This assumes that we have a certain amount of information to work with.

Getting all the details necessary to reproduce an issue, make a conclusion or even form a hypothesis about what's happening can take a fair amount of time. Our team is multiple orders of magnitude smaller than the RabbitMQ community. Please help others help you by providing a way to reproduce the behavior you're
observing and sharing as much relevant information as possible [on the list](https://groups.google.com/forum/#!forum/rabbitmq-users):

 * Server, client library and plugin (if applicable) versions used
 * Server logs
 * A code example or terminal transcript that can be used to reproduce
 * **Full** exception stack traces (not a single line message)
 * `rabbitmqctl status` (and, if possible, `rabbitmqctl environment` output)
 * Other relevant things about the environment and workload, e.g. OS/systemd logs, [a traffic capture](https://www.rabbitmq.com/amqp-wireshark.html), deployment tool logs, etc

Feel free to edit out hostnames and other potentially sensitive information.

When/if we have a complete enough understanding of what's going on, a recommendation will be provided or a new issues with more context will be filed.

Thank you.

**Comment by lukebakken** [comment]
This appears to be a valid issue using Erlang 21.2.2 and RMQ 3.7.10. Tracing the code now. Basically, `file_handle_cache:obtain/0` is used to increment the socket descriptor count and it doesn't seem to be happening.

**Comment by michaelklishin** [comment]
@lukebakken likely it's a side effect of the Ranch upgrade. I'm trying to compare.

**Comment by michaelklishin** [comment]
@essen can you think of something relevant that's changed in Ranch?

**Comment by michaelklishin** [comment]
Yes, this is a regression from `3.7.9`. Indeed this is before the [upgrade to Ranch 1.7](https://github.com/rabbitmq/rabbitmq-common/pull/284).

**Comment by lukebakken** [comment]
I believe this is due to deprecated `accept_ack` ([link](https://ninenines.eu/docs/en/ranch/1.7/guide/migrating_from_1.5/)), since that is where the socket count is incremented.
